Engines/Wine/Verbs/dotnet461/script.js

```diff
@@ -0,0 +1,57 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "net", "resource"]);
+include(["engines", "wine", "verbs", "luna"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "verbs", "dotnet40"]);
```

Comment:
I don't think all these includes are required. At least `dotnet40` and `dotnet452` should not be needed. Please include only used things.